protocol/errors.json

```diff
@@ -73,71 +69,54 @@
 	"40331": "unable to activate account due to placement constraint (incompatible environment)",
 	"40332": "unable to activate account due to placement constraint (incompatible site)",
 
-	/* 404 codes */
 	"40400": "not found",
 
-	/* 405 codes */
 	"40500": "method not allowed",
 
-	/* 410 codes */
 	"41001": "push device registration expired",
 
-	/* 422 codes */
 	"42200": "Unprocessable entity",
 
-	/* 429 codes */
 	"42910": "rate limit exceeded (nonfatal): request rejected (unspecified)",
 	"42911": "max per-connection publish rate limit exceeded (nonfatal): unable to publish message",
 	"42912": "there is a channel iteration call already in progress; only 1 active call is permitted to execute at any one time",
 	"42920": "rate limit exceeded (fatal)",
 	"42921": "max per-connection publish rate limit exceeded (fatal); closing connection",
 
-	/* 500 codes */
 	"50000": "internal error",
 	"50001": "internal channel error",
 	"50002": "internal connection error",
 	"50003": "timeout error",
 	"50004": "Request failed due to overloaded instance",
-	"50330": "request reached the wrong cluster; retry (used during dns changes for cluster migrations)",
+	"50005": "request reached the wrong cluster; retry (used during dns changes for cluster migrations)",
```

Comment:
Why has this changed? Searching for `50005` in the realtime code base, it's emitted [when the service is in lockdown](https://github.com/ably/realtime/blob/main/frontend/lib/service/rest.js#L37), which doesn't seem to match this description here.

Reply:
Oh, @SimonWoolf and I discussed over Slack and he thought it would be ok to rename as it's not always in use. See my additional detail on this commit's message: https://github.com/ably/ably-common/pull/74/commits/1d262b37386a8bc2a630011258d4a560d48eee02

Reply:
Hmm but I think we do set the code to 503 when we do the DNS switch over and I think that's part of the playbooks, just seemed odd to re-number it here amongst other changes. Anyway, If I see the code in a doc somewhere in the future I'll try and remember to change it 👍

Reply:
Sorry, yes, as Lewis says 50005 is already used by something else, but it never got added to ably-common, which is my fault. I'll do a pr to add it, but if you're renaming this then need to pick something else - I don't think 50006 is in use if you want something in the 500xx block

Reply:
Or just keep the code for this description the same and move it down in the file to after `50320`?

Reply:
cf comment https://github.com/ably/ably-common/pull/74#discussion_r653714585 - seems like the root of the problem is we currently have a bit of both of 'have functionally-related blocks of adjacent codes' and 'have codes be strongly tied to http status codes', and those are sometimes in tension

Reply:
Yes. @lmars, it may not be immediately obvious but I was also trying to gain some hold of the ranges / groups. See https://github.com/ably/ably-common/blob/feature/conform-protocol-json/protocol/README.md#error-code-ranges

Reply:
On balance, I think I'll add a commit to revert this code back to where it was, and as @lmars suggests position it correctly so everything's in sequence. I think it's going to also get its own range too - I'll also include that in the commit.

Reply:
https://github.com/ably/ably-common/pull/74/commits/e671455fbbcfa1b00821332c909f563a83808be6

@lmars please could you resolve this conversation if that fixes this for you?

Reply:
👍